server.js

```diff
@@ -0,0 +1,11 @@
+import http from 'http';
+import express from 'express';
+import socket_io from 'socket.io';
+
+let app = express();
+app.use(express.static('public'));
```

Comment:
changed to `build` since that is the directory `express` should serve from. All good on the homefront now 👍